var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -519,9 +520,13 @@ def install(self, spec, prefix):
         python('configure', '--prefix=%s' % prefix, *options)
 
         # PETSc has its own way of doing parallel make.
-        make('MAKE_NP=%s' % make_jobs, parallel=False)
+        make('V=1 MAKE_NP=%s' % make_jobs, parallel=False)
         make("install")
 
+        # archive configure.log make.log
+        shutil.copy('configure.log', os.path.join(self.prefix, '.spack'))
+        shutil.copy('make.log', os.path.join(self.prefix, '.spack'))
```

Comment:
To archive missing log files, can you add an `archive_files` property? You can check `cp2k` as an example for that, or have a look at the build system base classes:

https://github.com/spack/spack/blob/64c4fbd220508f86a1998a3677994578cfece2b3/lib/spack/spack/build_systems/autotools.py#L114-L120

Reply:
updated. Thanks!